src/utils/searchAlgorithm.js

```diff
@@ -0,0 +1,14 @@
+import { matchSorter } from 'match-sorter'
+import universityData from '../../data/university_data.json'
+
+const colleges = universityData.map((college) => {
+  return {
+    name: college.name,
+    address: college.address,
+  }
+})
+
+const searchAlgorithm = (input) => {
+  return matchSorter(colleges, input, { keys: ['name', 'address'] })
```

Comment:
Let's try to pass the whole `universityData` as the first argument.

Reply:
done!